internal/buildsign/ocpbuild/ocpbuildmanager.go

```diff
@@ -35,8 +32,6 @@ const (
 	StatusCreated    Status = "created"
 	StatusInProgress Status = "in progress"
 	StatusFailed     Status = "failed"
-
-	hashAnnotation = "kmm.node.kubernetes.io/last-hash"
```

Comment:
Why do we need to remove it? This is an internal constant of the buildsign package, it is not used anywhere else, why make it public?

Reply:
Just to make it more consistent with the [u/s code](https://github.com/kubernetes-sigs/kernel-module-management/blob/31aa5d0d89d63a809c8e7de83eae547666de285e/internal/constants/constants.go#L8). We can indeed stop exporting it but let's merge this one and fix it u/s first (to then be backported here).